rllib/env/single_agent_env_runner.py

```diff
@@ -608,8 +640,8 @@ def set_weights(
         # Only update the weigths, if this is the first synchronization or
         # if the weights of this `EnvRunner` lacks behind the actual ones.
         if weights_seq_no == 0 or self._weights_seq_no < weights_seq_no:
-            if isinstance(weights, dict) and DEFAULT_POLICY_ID in weights:
-                weights = weights[DEFAULT_POLICY_ID]
+            if isinstance(weights, dict) and DEFAULT_MODULE_ID in weights:
```

Comment:
Haha finally we get rid of the `DEFAULT_POLICY_ID` ghost that haunted us around the last year :D